packages/firestore/test/unit/local/query_engine.test.ts

```diff
@@ -67,25 +66,27 @@ const MISSING_LAST_LIMBO_FREE_SNAPSHOT = SnapshotVersion.min();
  * `getDocumentsMatchingQuery()` to detect index-free execution.
  */
 class TestLocalDocumentsView extends LocalDocumentsView {
-  expectIndexFreeExecution: boolean | undefined;
+  exceptFullCollectionScan: boolean | undefined;
```

Comment:
Typo: "exceptFullCollectionScan" should be "expectFullCollectionScan".

Reply:
Fixed